doc/source/user_guide/ug_case_settings.rst

```diff
@@ -239,6 +239,9 @@ grid_nml
    "``grid_file``", "string", "name of grid file to be read", "'unknown_grid_file'"
    "``grid_format``", "``bin``", "read direct access grid and kmt files", "``bin``"
    "", "``nc``", "read grid and kmt files", ""
+   "``grid_system``", "``B``", "use B grid structure with T at center and U at NE corner, "``B``"
+   "", "``C``", "use C grid structure with T at center, U at E edge, and V at N edge", ""
```

Comment:
We might need to be more specific in how we reference the grids.  In my current understanding, we will only implement the sea ice equations for the CD grid, but we will allow ocean coupling directly with either a C or CD grid.  Maybe we should use CD everywhere, since it's our main grid, and only mention C-grid in reference to coupling?

Reply:
I think we need to completely separate the solver and the coupling.  The solver may have B, C, CD options, whatever.  The coupling will have it's own set of options, maybe fields come in on the T grid, U grid, N grid, E grid, and that could be quite a mix and match for different coupled systems, different forcing files, etc.  It could even be that fields come in on a grid that is somehow not associated with the B, CD, etc grids.  We want a lot of flexibility for the coupling fields.

Reply:
Yes, agreed.  @JFLemieux73 can correct me on this, but I'm not expecting to have a purely C-grid solver.